app/page/template/content/preferences-devices.htm

```diff
@@ -32,7 +36,10 @@
                 </svg>
                 <div class="preferences-devices-card-info">
                   <div class="preferences-devices-model" data-bind="text: model" data-uie-name="preferences-device-active-model"></div>
-                  <div class="preferences-devices-id"><span data-bind="l10n_text: z.string.preferences_devices_id"></span><span data-bind="html: z.util.print_devices_id(id)" data-uie-name="preferences-device-active-id"></span></div>
+                  <div class="preferences-devices-id">
+                    <span data-bind="l10n_text: z.string.preferences_devices_id"></span>
+                    <span data-bind="foreach: z.util.zero_padding(id, 16).match(/.{1,2}/g)" data-uie-name="preferences-device-active-id"><span class="device-id-part" data-bind="text: $data"></span></span>
```

Comment:
Let's move this out of the HTML and into the view model

Reply:
Note to myself: This requires another variable (like `displayId`) in the ViewModel.